<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/menu_icon/menu_icon/icons/link_edit.png</normalon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabConnection">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="conn">
      <attribute name="title">
       <string>Connection 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ConnectionForm" name="connectionForm_1" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="conn2">
      <attribute name="title">
       <string>Connection 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkConn_2">
         <property name="text">
          <string>Enable connection 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ConnectionForm" name="connectionForm_2" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="conn3">
      <attribute name="title">
       <string>Connection 3</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkConn_3">
         <property name="text">
          <string>Enable connection 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ConnectionForm" name="connectionForm_3" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConnectionForm</class>
   <extends>QWidget</extends>
   <header>connectionform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
